x-pack/legacy/plugins/ml/server/models/job_validation/messages.js

```diff
@@ -24,15 +24,15 @@ export const getMessages = () => {
         },
       }),
       url:
-        'https://www.elastic.co/guide/en/elastic-stack-overview/{{version}}/ml-configuring-aggregation.html',
+        'https://www.elastic.co/guide/en/machine-learning/{{version}}/ml-configuring-aggregation.html',
```

Comment:
You will need to rebase against the changes I made in https://github.com/elastic/kibana/pull/53576, and edit all the links in this file to the previous Create anomaly detection jobs page. So the `createJobsDocsUrl` field I added in my PR needs to be amended to `https://www.elastic.co/guide/en/machine-learning/{{version}}/create-jobs.html`.

Reply:
Thanks @peteharverson !